src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/controller/RecallController.kt

```diff
@@ -256,15 +259,19 @@ fun BookRecallRequest.toRecall(userUuid: UserId): Recall {
     now,
     this.firstName,
     this.middleNames,
-    this.lastName
+    this.lastName,
+    this.croNumber,
+    this.dateOfBirth
   )
 }
 
 data class BookRecallRequest(
   val nomsNumber: NomsNumber,
   val firstName: FirstName,
   val middleNames: MiddleNames?,
-  val lastName: LastName
+  val lastName: LastName,
+  val croNumber: CroNumber,
```

Comment:
I guess this has been mandatory before, just not explicitly here?

Reply:
hmm, good point. I do think we found some examples in dev where this wasn't populated. but we'll just skip those 😬